webservices/resources/legal.py

```diff
@@ -105,7 +105,7 @@ def get(self, q='', from_hit=0, hits_returned=20, type='all', **kwargs):
                 .sort("sort1", "sort2")
 
             if type == 'advisory_opinions':
-                query = apply_ao_specific_query_params(query, **kwargs)
+                query = apply_ao_specific_query_params(query, q, **kwargs)
```

Comment:
Ah okay, thanks!  I thought this was the case, but the `query` parameter was throwing me off.